src/StatCache.cc

```diff
@@ -59,7 +59,7 @@ int StatCache::stat(const char *path, struct stat *st)
 		}
 		statMap.erase(iter);                            // Remove stale entry
 	}
-	CacheEntry entry{};                               // Make a new entry
+	CacheEntry entry;                                 // Make a new entry
```

Comment:
Which compiler gives this warning? To my knowledge, this is explicitly how structs are supposed to be zero-initialized in C++11.